tests/by-util/test_free.rs

```diff
@@ -27,6 +31,19 @@ fn test_free_wide() {
     assert!(!result.stdout_str().contains("buff/cache"));
 }
 
+#[test]
+fn test_free_total() {
+    let result = new_ucmd!().arg("-t").succeeds();
+    assert_eq!(result.stdout_str().lines().count(), 4);
+    assert!(result.stdout_str().lines().collect::<Vec<&str>>()[3].starts_with("Total:"))
+}
+
+#[test]
+fn test_free_count() {
+    let result = new_ucmd!().args(&["-c", "2", "-s", "0"]).succeeds();
```

Comment:
This test is about `-c`, so I would omit the `-s` as it's not necessary.
```suggestion
    let result = new_ucmd!().args(&["-c", "2"]).succeeds();
```

Reply:
i did that so i have to wait less 😂